csr/csr.go

```diff
@@ -244,7 +259,12 @@ func getNodeCSRV1Beta1(ctx context.Context, client kubernetes.Interface, nodeNam
 
 	for _, item := range csrList.Items {
```

Comment:
Why we aren't using a watch for CSR like in V1 API?

Reply:
the beta path is legacy and we don't have any clusters using it, didn't want to invest much time into it and rather remove it completely in the future